quinn-proto/src/connection/streams/recv.rs

```diff
@@ -215,14 +215,18 @@ impl<'a> Chunks<'a> {
         streams: &'a mut StreamsState,
         pending: &'a mut Retransmits,
     ) -> Result<Self, ReadableError> {
-        let entry = match streams.recv.entry(id) {
+        let mut entry = match streams.recv.entry(id) {
             Entry::Occupied(entry) => entry,
             Entry::Vacant(_) => return Err(ReadableError::UnknownStream),
         };
 
-        let mut recv = match entry.get().stopped {
+        let mut recv = match entry
+            .get_mut()
+            .get_or_insert_with(|| Box::new(Recv::new(streams.stream_receive_window)))
+            .stopped
+        {
             true => return Err(ReadableError::UnknownStream),
-            false => entry.remove(),
+            false => entry.remove().unwrap(), // this can't fail at this point
```

Comment:
Best practice is to say *why* it can't fail (due to `get_or_insert_with` above).